src/migrations/1610030345948_bns-subdomain.ts

```diff
@@ -85,6 +85,7 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
   pgm.createIndex('subdomains', 'latest');
   pgm.createIndex('subdomains', 'atch_resolved');
   pgm.createIndex('subdomains', 'resolver');
+  pgm.createIndex('subdomains', 'owner');
```

Comment:
We need to create this index in a new migration file, now that production deployments are trying to avoid database wipes on version upgrades.